### ISSUE TITLE
[Bug?]: Avatar & Icon: missing aria-label for iconOptions & icon API

### ISSUE THREAD

**Issue by Rykus0** [pluralsight/tva#515]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

aria-label is only applied to the `label` element of the Avatar, which is not used by the icon (or image, but that is secondary given the `alt` attribute, but more on that in the next section)

### Expected behavior 🤔

The avatar should always have an accessible descriptor by which it is described or labelled.  

Some options to accomplish this are:
1. Move the aria-label to the wrapper element and always have it present
2. Adjust the iconOptions so that the icon is not hidden and has an aria-label configured

**Related:** The image's alt attribute could be the initials so that if the image does not display for any reason, the fallback display is still compliant with the visual design.  Placing the aria-label on the wrapper would support this, in which case the image would be hidden from screen readers and similar assistive technology.

Moving the aria-label to the wrapper also supports allowing for more flexible wrapper elements, so that when the focus is applied, the label is recognized by the assistive tech.

### Steps to reproduce 🕹

Steps:

1.
2.
3.
4.
